### ISSUE TITLE
Add an examples directory

### ISSUE THREAD

**Issue by dtolnay** [djc/askama#74]
It would be helpful to have a copy of the "How to get started" readme section as a fully laid out [examples](https://doc.rust-lang.org/cargo/reference/manifest.html#examples) directory where someone could just drop in and do `cargo run` and see everything work.

**Comment by tobias47n9e** [comment]
I put a minimal askama + rocket example here in case anyone is looking for just that: https://gitlab.com/tobias47n9e/rocket_with_askama_example

**Comment by Th3Whit3Wolf** [comment]
Is this still an issue? I know repos like [OrbTK](https://github.com/redox-os/orbtk) where you can compile examples with `cargo run --example widgets --release`.

Another option would be to make a repo just for askama examples or to use write examples in markdown with mdbook and render them with gh-pages.

**Comment by djc** [comment]
Yes, it is still an issue. A separate crate inside this repo could work, I suppose.
